python/pyspark/tests/test_rdd.py

```diff
@@ -60,15 +60,12 @@ def test_sum(self):
         self.assertEqual(6, self.sc.parallelize([1, 2, 3]).sum())
 
     def test_to_localiterator(self):
-        from time import sleep
         rdd = self.sc.parallelize([1, 2, 3])
         it = rdd.toLocalIterator()
-        sleep(5)
```

Comment:
This sleep is unnecessary. `rdd.toLocalIterator` makes the socket connection and iterating starts reading from the pyspark serializer.

Reply:
I like less sleeps in the code <3